api/v2/views/base.py

```diff
@@ -175,11 +175,17 @@ def perform_update(self, serializer):
                 instance = serializer.save(end_date=timezone.now(),
                                            membership=membership)
             else:
-                instance = serializer.save(end_date=timezone.now())
+                if self.request._method == "PATCH":
+                    instance = serializer.save(status=StatusType.objects.get(id=serializer.initial_data['status']))
+                else:
+                    instance = serializer.save(end_date=timezone.now())
```

Comment:
I don't know if we should be saving the 'end_date' every time here. That is not the correct behavior (Although i see above thats what was done previously).. @prosif Can you make a note in JIRA to address this in a different commit/PR?